source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -4,6 +4,12 @@ Access Data From a Cursor
 
```

Comment:
Issue (Out of Scope but should be done):  This page does not contain an "Overview" section and does not define the term "Cursor". 

As this edit is definitely out of scope, I will not press any further if you choose not to pursue it.

If you choose not to do this edit I will break it out into a separate ticket.

Reply:
In a synchronous call, we agreed to capture this item in another ticket.